manifests/4.5/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -0,0 +1,275 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterlogforwarders.logging.openshift.io
+spec:
+  group: logging.openshift.io
+  names:
+    kind: ClusterLogForwarder
+    listKind: ClusterLogForwarderList
+    plural: clusterlogforwarders
+    singular: clusterlogforwarder
+  scope: cluster
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: ClusterLogForwarder is the schema for the `clusterlogforwarder`
+        API.
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: ClusterLogForwarderSpec defines the desired state of ClusterLogForwarder
+          properties:
+            outputs:
+              description: Outputs are named destinations for log messages.
+              items:
+                description: Output defines a destination for log messages.
+                properties:
+                  elasticsearch:
+                    type: object
+                  fluent:
+                    type: object
+                  insecure:
+                    description: Insecure must be true for intentionally insecure
+                      outputs. Has no function other than a marker to help avoid configuration
+                      mistakes.
+                    type: boolean
+                  name:
+                    description: Name used to refer to the output from a `pipeline`.
+                    type: string
+                  secretRef:
+                    description: "SecretRef refers to a `secret` object for secure
+                      communication. \n Client-authenticated TLS is enabled if the
+                      secret contains keys `tls.crt`, `tls.key` and `ca.crt`. Output
+                      types with password authentication will use keys `password`
+                      and `username`, not the exposed 'username@password' part of
+                      the `url`."
+                    properties:
+                      name:
+                        description: Name is unique within a namespace to reference
+                          a secret resource.
+                        type: string
+                      namespace:
```

Comment:
This is the k8s.io.api.core.v1.SecretReference, which seems to be a very common way to refer to a secret in an API. I can replace it with a string, but I wasn't confident we'll never need the namespace, in particular since the ClusterLogForwarder is deployed in cluster scope.  What do you think?